files/docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Image for the web service (httpd), for celery worker see Dockerfile.worker
```

Comment:
How are these files different from the ones in the repository root? Is the build configuration need to be updated to use these files?

Reply:
this change require hooks directory in Dockerfile location. To avoid messy project root dir i have decided to move Dockerfiles and hooks to dedicated location. 

Because of dockerhub autobuilds depends on locations of Dockerfile's, this is required to perform "smooth"  transition. Once this PR is merged, i will configure dockerhub to consume Dockerfiles in new location, test it, document it  and then remove the ones from project root

Reply:
Would it make sense to add a short README in files/docker to explain the usage? I had to read the [docs](https://docs.docker.com/docker-hub/builds/advanced/#custom-build-phase-hooks) twice to understand how will this work :joy_cat:

Reply:
sure - will update docs

Reply:
documented in deployment as there is all related docu and added README.md with link to docu (https://github.com/packit-service/deployment/pull/108)